hack/python-sdk/gen-sdk.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
```

Comment:
when should this script be run, should we run it as part of `make generate`?

Reply:
yep i think we should, added